content/src/anytime-action-license-renewals/health-care-services-firm-renewal.md

```diff
@@ -1,15 +1,15 @@
 ---
 notesMd: ""
-urlSlug: health-care-services-firm-renewal
-renewalEventDisplayName: Health Care Service Firm Renewal Deadline
-expirationEventDisplayName: Health Care Service Firm Expiration
+filename: health-care-services-firm-renewal
```

Comment:
Boulder: the goal of this ticket is to add the license status component to the `[licenseUrlSlug]` page. This page renders the content living within `content/src/licenses`. The user is able to access the calendar license page by clicking the renewal/expiration link on the filings calendar. Alot of the existing functionality you added can be removed because it's already existing (i.e. mgmt/search)

Reply:
One approach is to add the existing component to the [licenseUrlSlug] page and create tests to validate the data renders.  All the other logic is not necessary because it's working. You may want to refactor the content/src/licenses to something like content/src/licenseCalendarEvents so it's more explicit.